infrastructure/helm-chart/charts/apps/charts/sources-google-events-router/templates/deployment.yaml

```diff
@@ -42,6 +42,16 @@ spec:
             configMapKeyRef:
               name: kafka-config
               key: KAFKA_COMMIT_INTERVAL_MS
+        - name: GOOGLE_SA_FILE
+          valueFrom:
+            configMapKeyRef:
+              name: sources-google
+              key: GOOGLE_SA_FILE
+        - name: GOOGLE_PARTNER_KEY
+          valueFrom:
+            configMapKeyRef:
+              name: sources-google
+              key: GOOGLE_PARTNER_KEY
```

Comment:
Dependency which is not required, but so that the events-router doesn't start upfront (same for Twilio).